### PR TITLE
Added try/catch for failed merge checks

### PR DESCRIPTION
**Pull request by orangejenny**
Band-aid for this code not handling forks well.

### CODE REVIEW

**Review by gherceg — APPROVED** [review]
Looks like a step in the right direction. Approving but might be good for others to weigh in. Thanks!

**Comment by orangejenny** [comment]
Updated to try actually fixing the issue, but haven't quite figured out how to test.

File: git_build_branch/gitutils.py [hunk 1]
```diff
@@ -65,7 +65,14 @@ def git_check_merge(branch1, branch2, git=None):
     Thanks to http://stackoverflow.com/a/501461/240553
 
     """
+    def format_branch(branch):
+        if ":" in branch:
+            branch = branch.replace(":", "/")
+        return branch
```
Comment (millerdev) [inline git_build_branch/gitutils.py:71]: nit
```suggestion
        return branch.replace(":", "/")
```

**Review by millerdev — APPROVED** [review]
Looks like [colon is not a valid character in a branch name](https://wincent.com/wiki/Legal_Git_branch_names), so I guess it's safe.

**Pull request merged**